db/migrate/20231128103215_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- You have done well so far, but according to the previous reviewer please ensure that you correctly set up your database to match the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), your likes table should reference the `user` table and `posts` table, please fix this 👍 👍